apps/profilers/src/bin/block-serde-profiler.rs

```diff
@@ -31,7 +30,10 @@ fn main() -> anyhow::Result<()> {
 }
 
 fn cpu_profile_serialization() -> anyhow::Result<ExternalTransition> {
-    Ok(ExternalTransition::try_decode_binprot(BLOCK_BYTES)?)
+    let mut de = bin_prot::Deserializer::from_reader(BLOCK_BYTES);
+    let et: <ExternalTransition as SerializationTypeAnnotation>::BinProtType =
```

Comment:
This is very verbose but would be just `<ExternalTransition as SerializationType>::BinProt` with suggest renaming

Reply:
fixed